src/__tests__/React/ReactUI.test.js

```diff
@@ -0,0 +1,87 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import { Provider } from "react-redux";
+import thunk from "redux-thunk";
+import fetchMock from "fetch-mock";
+import MockDate from "mockdate";
+import { createStore, applyMiddleware } from "redux";
+import fs from 'fs';
+import App from '../../App';
+import reducer from "../../redux/cases/cases"
+
+
+const getMockedData = () => {
+  const data = fs.readFileSync('./src/__tests__/React/mockData.json', 'utf8')
+  return JSON.parse(data);
+}
+
+const createTestStore = () => {
+  const store = createStore(
+    reducer,
```

Comment:
- [ ] All tests also fail to run they throw an error: `Cannot find module 'node-fetch' from 'node_modules/fetch-mock/cjs/server.js'`. Kindly check and fix it.